packages/lwc-language-server/src/resources/convert.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+
+function getComponentLibraryLink(name) {
+    return '[View in Component Library](https://developer.salesforce.com/docs/component-library/bundle/' + name + ')';
+}
+
+function getHover(tag) {
+    let retVal = tag.description + '\n' + getComponentLibraryLink(tag.name) + '\n### Attributes\n';
+
+    for (const info of tag.attributes) {
+        retVal += getAttributeMarkdown(info);
+        retVal += '\n';
+    }
+
+    return retVal;
+}
+
+function getAttributeMarkdown(attribute) {
+    if (attribute.name && attribute.type && attribute.description) {
+        return '* **' + attribute.name + '**: *' + attribute.type + '* ' + attribute.description;
+    }
+
+    if (attribute.name && attribute.type) {
+        return '* **' + attribute.name + '**: *' + attribute.type + '*';
+    }
+
+    if (attribute.name) {
+        return '* **' + attribute.name + '**';
+    }
+
+    return '';
+}
+
+// read old file
+const f = fs.readFileSync('lwc-standard.json', { encoding: 'utf-8' });
```

Comment:
`lwc-standard.json` will keep growing as time goes by. Is it worth switching to nodejs streams for read and write ?